routes/api.js

```diff
@@ -8,22 +8,37 @@ const AuthController = require('../app/Controllers/AuthController.js');
 const LanguagePackageController = require('../app/Controllers/LanguagePackageController.js');
 const GroupController = require('../app/Controllers/GroupController.js');
 const VocabularyController = require('../app/Controllers/VocabularyController.js');
+const QueryController = require('../app/Controllers/QueryController.js');
 const DocsController = require('../app/Controllers/DocsController');
 
 const router = express.Router();
 
 // AUTH
-router.post('/auth/register', AuthController.register);
-router.post('/auth/login', AuthController.login);
+router.post('/user/register', AuthController.register);
+router.post('/user/login', AuthController.login);
 router.get('/auth', ProtectMiddleware, AuthController.profile);
```

Comment:
could we change this also to `/user` ?

Reply:
I have changed it to /user in another branch already